server/data_server.c

```diff
@@ -101,13 +101,40 @@ static const struct data_capture *data_capture;
 static struct timespec pcap_arm_ts;
 /* PCAP becomes armed & enabled timestamp */
 static struct timespec pcap_start_ts;
+static bool pcap_hw_ts_offset_ns_valid;
+static int64_t pcap_hw_ts_offset_ns;
```

Comment:
So should this be a float?

Reply:
For future reference, It's integer to keep code consistent with libc timespec structure